Sources/Purchasing/StoreKit2/StoreKit2TransactionFetcher.swift

```diff
@@ -73,6 +76,29 @@ final class StoreKit2TransactionFetcher: StoreKit2TransactionFetcherType {
         }
     }
 
+    @available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *)
+    var receipt: StoreKit2Receipt {
```

Comment:
This isn't tested?

Reply:
Its tested (see `testGeneratesReceipt`) but I wonder how Codecov generated coverage, and if it is only from the latest run test (e.g. if it the last run test was iOS 14, this method would not have been tested)